sources/packages/backend/libs/services/src/sfas/sfas-application.service.ts

```diff
@@ -73,6 +75,48 @@ export class SFASApplicationService extends DataModelService<SFASApplication> {
     return +(total?.sum ?? 0);
   }
 
+  /**
+   * Fetch the SFAS full time application for the student and the latest
+   * application which has the end date within 730 days.
+   * @param studentId student id.
+   * @returns SFASSignedMSFAA which contains the SFAS Signed MSFAA
+   * and latest application end date.
+   */
+  async getIndividualFullTimeApplicationByIndividualId(
+    studentId: number,
+  ): Promise<SFASSignedMSFAA> {
+    const twoYearsAgo = new Date();
+    twoYearsAgo.setDate(twoYearsAgo.getDate() - MAX_MSFAA_VALID_DAYS);
```

Comment:
Can we replace lines 88 and 89 with the below?
Also the name `twoYearsAgo` represents what the constant is right now but if it changes in the future the name will lose its meaning. See below a suggestion.
```ts
const minMSFAAValidDate = addDays(-MAX_MSFAA_VALID_DAYS);
```